lib/install/phases/bootstrap.go

```diff
@@ -220,6 +220,8 @@ func (p *bootstrapExecutor) configureSystemDirectories(ctx context.Context) erro
 		filepath.Join(stateDir, "secrets"),
 		filepath.Join(stateDir, "backup"),
 		filepath.Join(stateDir, "logrange"),
+		filepath.Join(stateDir, "monitoring", "prometheus-db"),
```

Comment:
nit: but why not simply `prometheus` and `alertmanager` - are we planning to have more directories under monitoring prefixed with `prometheus`/`alertmanager`?

Reply:
Yeah, I would just name "prometheus" and "alertmanager".

Reply:
```
Implementation

prometheus-db and alertmanager-db are hardcoded subPath for volume mounts in prometheus-operator.```

Reply:
I would like to create only `monitoring` path and leave it like that. But prometheus operator makes volume mounts with hardcoded paths `alertmanager/prometheus-db` and if directory does not exist Kubernetes creates it with the permissions of kubelet, which is root/root. In that case I can't write in them because those containers are running with service-user. This is why I explicitly create `alertmanager-db/prometheus-db`.

Reply:
Ok - lgtm then.